### DISCUSSION TITLE
how to create an app in Alchemy now？

### DISCUSSION THREAD

**Discussion by Ehenglin77** [Cyfrin/foundry-full-course-cu#2733] [Q&A]
It looks different from the tutorial

**Comment by pollersan** [comment]
Hello,

You can create it like this:

![image](https://github.com/user-attachments/assets/a49c932c-0542-4ca9-ad80-71039414651d)
![image](https://github.com/user-attachments/assets/dbcc792e-9a3f-4930-b665-dcc76e1fd45a)
![image](https://github.com/user-attachments/assets/6df4d6ee-d28d-4219-bd2c-e8ec667c70c4)
![image](https://github.com/user-attachments/assets/a530a931-3358-4c98-b2f6-593cf145bf91)

**Reply by Ehenglin77** [reply]
thank you

**Comment by EngrPips** [comment]
Interesting, Thanks for the heads-up @pollerpan

**Comment by Ehenglin77** [comment]
There is a new problem again.
forge script script/deploySimpleStorage.s.sol --rpc-url=$Sepolia_RPC_URL --private-key=$PRIVATE_KEY --broadcast
Error: 
Internal transport error: Connection refused (os error 111) with /home/user_eyon/solidity/test/

whats the problem here？

**Reply by EngrPips** [reply]
I am unsure but maybe try `rpc-url` from another provider such as `infura` and `Quicknode` and see if that helps.

**Reply by Ehenglin77** [reply]
I found where the problem is. I forgot to save the .env file, so the 'source .env' command didn't take effect."

**Reply by EngrPips** [reply]
Interesting, I am glad you figured that out.
